src/main/webapp/js/userMapData.json

```diff
@@ -1382,14 +1505,17 @@
       "University of Massachusetts",
       "University of Massachusetts Amherst",
       "University of Massachusetts Boston",
+      "University of Massachusetts, Amherst",
       "University of Memphis",
       "University of Michigan",
       "University of Minnesota",
+      "University of Minnesota Twin Cities",
       "University of Minnesota, Twin Cities",
       "University of Mississippi",
       "University of Missouri",
       "University of Nebraska",
       "University of Nebraska-Lincoln",
+      "University of Nebraska–Lincoln",
```

Comment:
Sorry about the obvious repeats. I had removed them in one of my previous commits but forgot to remove them again after I reverted that commit. I will make the changes as soon as possible.